doc/source/user_guide/ug_case_settings.rst

```diff
@@ -2,8 +2,8 @@
 
 .. _case_settings:
 
-Case Settings
-=====================
+Model Namelist, CPPs, and Other Settings
+==========================================
```

Comment:
We have one instance of the code itself referencing the "Case Settings" section of the documentation:

https://github.com/CICE-Consortium/CICE/blob/a5bb4f9a0c180e325e2a5480832f588dbfdd25ec/cicecore/cicedyn/general/ice_init.F90#L1717-L1720

 So I think we should at least keep that as part of the section title, maybe something like:

> Case Settings: Model Namelist, CPPs, and Other Settings